.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,57 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
I suppose this is not really part of the PR, always a bit confused with InteeliJ files..

Reply:
> I suppose this is not really part of the PR

It was checked-in on purpose, in a different commit! 😉 

> always a bit confused with InteeliJ files..

Nowadays, `.gitignore` files have been tweaked so that only files that matter are checked-in (e.g. code style). If there are changes, it's very likely JetBrains decided to change some stuff (new naming for specific properties, how files are organized, etc.). So in most cases, it's safe to commit the updates but going through it is always good: in case of that recent update, they pushed for Java 8 refactoring, so I made sure `Anonymous2MethodRef`, `Convert2Lambda`, etc. is set to `false`.